com.eminfedar.vaktisalah.json

```diff
@@ -0,0 +1,31 @@
+{
+    "id": "com.eminfedar.vaktisalah",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15",
+    "sdk": "org.kde.Sdk",
+    "command": "vaktisalah",
+    "finish-args": [
+        "--share=network",
+        "--socket=pulseaudio",
+        "--share=ipc",
+        "--socket=x11"
+    ],
+
+    "modules": [
+        {
+            "name": "vaktisalah",
+            "buildsystem": "qmake",
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/eminfedar/vaktisalah.git",
+                    "commit": "c1534ea729f66d2073914b41b082493c6d4ff8fb"
+                }
+            ],
+            "build-commands": [
```

Comment:
this won't work since buildsystem is `qmake`. What you are looking for is `post-install`. Don't you need to install the appdata too? It needs to be listed in the sources here as well.

Reply:
Oh, but it worked when I tried and I was confused too :)

Actually it would be very nice if a complete tutorial about preparing a flatpak package. Informations on documents are so seperate and not has detailed information. It is hard to find if I should put something to package. For example how can I achieve the "copying desktop & icon files" on qmake only build system? It is a little bit complicated for beginners.

Thanks for your good informative comments.

Reply:
I have changed `build-commands` to `post-install` and it seems working.